Update seat-management.mdx (#22360)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/users/seat-management.mdx

### Enable seat expiration

> Cloudflare Zero Trust can automatically remove any user who does not log in to an Access application or whose device does not show any Gateway activity for the specified period (between one month and one year). To determine if a user will be removed, the system looks for any authentication events as well as the user's "last seen value" for all their associated devices, if both of those are outside the expiration window, their seat will be removed. This process occurs once daily for an account. These users will no longer count against your number of seats.

Reviewer: ```suggestion
Cloudflare Zero Trust can automatically remove any user who does not log in to an Access application or whose device does not show any Gateway activity for the specified period (between one month and one year). To determine if a user will be removed, Cloudflare looks for any authentication events and checks the **Last seen** value for all of the user's devices. If both of those are outside the expiration window, the user will be removed and will no longer count against your number of seats. This process occurs once daily for an account.
```
